packages/office-addin-dev-settings/src/sideload.ts

```diff
@@ -334,6 +330,10 @@ export async function sideloadAddIn(manifestPath: string, appType: AppType, app?
 
   if (sideloadFile) {
     if (app == OfficeApp.Outlook) {
+      // check to see if Outlook exe path contains spaces and escape the path if it does.
+      if (sideloadFile.indexOf(' ') >= 0){
+        sideloadFile = `"${sideloadFile}"`;
```

Comment:
Do you have to check if there's a space in order to add double-quotes?  This is fine, but just wondering.

Reply:
yes that's correct